forte/mrdsrg-spin-integrated/master_mrdsrg.cc

```diff
@@ -250,17 +251,56 @@ void MASTER_DSRG::fill_density() {
 void MASTER_DSRG::init_fock() {
     outfile->Printf("\n    Filling Fock matrix from ForteIntegrals ......... ");
     Fock_ = BTF_->build(tensor_type_, "Fock", spin_cases({"gg"}));
-    Fock_.iterate([&](const std::vector<size_t>& i, const std::vector<SpinType>& spin, double& value) {
-        if (spin[0] == AlphaSpin) {
-            value = ints_->get_fock_a(i[0], i[1]);
-        } else {
-            value = ints_->get_fock_b(i[0], i[1]);
-        }
-    });
+
+    bool JKSafe = foptions_->get_bool("EMBEDDING_JKFOCK");
```

Comment:
Is JK "unsafe" in some cases? What's the danger?

Reply:
The problem is that I write dressed integral valus directly to the integral class, while the DSRG code will try to build the integrals from JK builder, which will give the undressed integrals. So I have to bypass it.